.travis.yml

```diff
@@ -23,11 +23,9 @@ before_install:
 # This is only run before integrationTests job
 # To ensure docker containers are fully up and running we sleep 20s
```

Comment:
Since the sleep time has increased, update the comments to match :)

Reply:
Done